src/lib/merkle_ledger/intf.ml

```diff
@@ -332,18 +332,16 @@ module Ledger = struct
       t -> init:'accum -> f:(Addr.t -> 'accum -> account -> 'accum) -> 'accum
 
     (** the set of [account_id]s are ledger elements to skip during the fold,
-      because they're in a mask
-  *)
+        because they're in a mask *)
```

Comment:
Prefer closing doc-comment on new line, per the style guide.

Reply:
Done